package.json

```diff
@@ -24,6 +24,7 @@
     "test-filter-parsing-errors": "./node_modules/.bin/tape tests/test-filter-parsing-errors.js | ./node_modules/.bin/faucet"
```

Comment:
Good catch! I had no idea that these test scripts were defined in `package.json`.